aioodbc/connection.py

```diff
@@ -172,18 +185,23 @@ async def execute(self, sql: str, *args) -> Cursor:
                 await self.close()
             raise
 
-    def getinfo(self, type_):
+    def getinfo(self, type_: int) -> asyncio.Future[Any]:
```

Comment:
Lets make this proper coroutine (`async def`) so signature is not polluted by `asyncio.Future`

Reply:
Thanks for this feedback. I just realized the official documentation makes that recommendation as well ([link](https://docs.python.org/3/library/asyncio-future.html#asyncio.Future)).